Description: Ensure source/target is specified so backwards
 compatible bytecode is generated during the build process.
Author: James Page <james.page@ubuntu.com>
Forwarded: no

Index: libbluray-0.2.2/src/libbluray/bdj/build.xml
===================================================================
--- libbluray-0.2.2.orig/src/libbluray/bdj/build.xml	2010-07-19 10:54:58.000000000 +0100
+++ libbluray-0.2.2/src/libbluray/bdj/build.xml	2012-08-06 11:44:21.099044620 +0100
@@ -15,7 +15,8 @@
 
     <target name="compile" depends="init"
             description="compile the source " >
-        <javac srcdir="${src}" destdir="${build}" debug="yes" />
+        <javac srcdir="${src}" destdir="${build}" debug="yes"
+               source="1.5" target="1.5"/>
     </target>
 
     <target name="dist" depends="compile"
